pom.xml

```diff
@@ -374,7 +401,7 @@
           <dependency>
             <groupId>org.codehaus.groovy</groupId>
             <artifactId>groovy-eclipse-batch</artifactId>
-            <version>2.4.3-01</version>
+            <version>2.4.16-03</version>
```

Comment:
This is to fix
```
Ignoring GroovyBugError since it is likely caused by earlier issues.  Ignored problem is 'BUG! exception in phase 'semantic analysis' in source unit 'C:\sonatype\jenkins-nexus-platform-plugin\src\main\java\org\sonatype\nexus\ci\nxrm\Coordinate.groovy' Attempt to change an immutable Groovy class: java.lang.Class'
Internal Groovy Error --- exception in phase 'semantic analysis' in source unit 'C:\sonatype\jenkins-nexus-platform-plugin\src\main\java\org\sonatype\nexus\ci\nxrm\Coordinate.groovy' Attempt to change an immutable Groovy class: java.lang.Class
BUG! exception in phase 'semantic analysis' in source unit 'C:\sonatype\jenkins-nexus-platform-plugin\src\main\java\org\sonatype\nexus\ci\nxrm\Coordinate.groovy' Attempt to change an immutable Groovy class: java.lang.Class
        at org.codehaus.groovy.ast.ImmutableClassNode.setGenericsTypes(ImmutableClassNode.java:92)
```